types/types.go

```diff
@@ -42,6 +42,16 @@ type State struct {
 	Hangs   InputCorpus
 }
 
+type Target struct {
```

Comment:
We use these in both `client` and `server` so I thought putting them here made sense.